hooks/useLayout.ts

```diff
@@ -0,0 +1,15 @@
+import { useContext } from 'react';
+import LayoutContext from '@/stores/Layout.context';
+
+const useLayoutMumble = () => {
```

Comment:
I would either rename file to useLayoutMumble or the function to useLayout to make the naming consistent